core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -380,6 +392,12 @@ private[spark] class SparkSubmitArguments(args: Seq[String], env: Map[String, St
         |  --name NAME                 A name of your application.
         |  --jars JARS                 Comma-separated list of local jars to include on the driver
         |                              and executor classpaths.
+        |  --maven                     Comma-separated list of maven coordinates of jars to include
+        |                              on the driver and executor classpaths. Will search the local
+        |                              maven repo, then maven central and any additional remote
+        |                              repositories given by --maven_repos.
```

Comment:
Instead of taking one parameter with a list of all Maven packages, we might want to allow separate packages to be passed with separate `--maven` args. Dunno, @pwendell / @mengxr what do you think? It's just going to be annoying for people to write a giant comma separated string. Same with repos actually.